agentops/__init__.py

```diff
@@ -44,102 +31,111 @@ def init(
+    instrument_llm_calls: Optional[bool] = None,
+    auto_start_session: Optional[bool] = None,
     inherited_session_id: Optional[str] = None,
-    skip_auto_end_session: Optional[bool] = False,
+    skip_auto_end_session: Optional[bool] = None,
 ) -> Union[Session, None]:
     """
     Initializes the AgentOps singleton pattern.
 
     Args:
-
         api_key (str, optional): API Key for AgentOps services. If none is provided, key will
             be read from the AGENTOPS_API_KEY environment variable.
         parent_key (str, optional): Organization key to give visibility of all user sessions the user's organization. If none is provided, key will
             be read from the AGENTOPS_PARENT_KEY environment variable.
         endpoint (str, optional): The endpoint for the AgentOps service. If none is provided, key will
             be read from the AGENTOPS_API_ENDPOINT environment variable. Defaults to 'https://api.agentops.ai'.
         max_wait_time (int, optional): The maximum time to wait in milliseconds before flushing the queue.
-            Defaults to 30,000 (30 seconds)
+            Defaults to 5,000 (5 seconds)
         max_queue_size (int, optional): The maximum size of the event queue. Defaults to 100.
-        tags (List[str], optional): Tags for the sessions that can be used for grouping or
-            sorting later (e.g. ["GPT-4"]).
+        tags (List[str], optional): [Deprecated] Use `default_tags` instead.
+        default_tags (List[str], optional): Default tags for the sessions that can be used for grouping or sorting later (e.g. ["GPT-4"]).
         instrument_llm_calls (bool): Whether to instrument LLM calls and emit LLMEvents.
         auto_start_session (bool): Whether to start a session automatically when the client is created.
         inherited_session_id (optional, str): Init Agentops with an existing Session
-        skip_auto_end_session (optional, bool): Don't automatically end session based on your framework's decision-making (i.e. Crew determining when tasks are complete and ending the session)
+        skip_auto_end_session (optional, bool): Don't automatically end session based on your framework's decision-making
+            (i.e. Crew determining when tasks are complete and ending the session)
     Attributes:
     """
-    logging_level = os.getenv("AGENTOPS_LOGGING_LEVEL")
-    log_levels = {
-        "CRITICAL": logging.CRITICAL,
-        "ERROR": logging.ERROR,
-        "INFO": logging.INFO,
-        "WARNING": logging.WARNING,
-        "DEBUG": logging.DEBUG,
-    }
-    logger.setLevel(log_levels.get(logging_level or "INFO", "INFO"))
-
-    c = Client(
+    Client().unsuppress_logs()
+    if Client().is_initialized:
+        return logger.warning("AgentOps has already been initialized")
+
+    if tags is not None:
+        logger.warning("The 'tags' parameter is deprecated. Use 'default_tags' instead")
+        if default_tags is None:
+            default_tags = tags
```

Comment:
we have tests for add_tags and set_tags rn. it might be worth writing some for default tags too, they're pretty simple